command/agent/config/test-fixtures/config-cache-auto_auth_token-no-auto_auth.hcl

```diff
@@ -0,0 +1,12 @@
+pid_file = "./pidfile"
+
+cache {
+  use_auto_auth_token = true
```

Comment:
I don't think we should allow such a config to work without throwing some kind of error, the reason being that if ` use_auto_auth_token` is enabled, there needs to be a method for the agent to use auto-auth to get a token, hence the need for an auto-auth config block.

Reply:
100%, this was a think-o. After re-reading the https://www.vaultproject.io/docs/agent/caching#using-auto-auth-token, an `auto_auth` method would have to be established in order for an `auto_auth` token to be acquired, else, where would it come from? 😄